geode-core/src/test/java/org/apache/geode/internal/net/BufferPoolTest.java

```diff
@@ -22,12 +22,18 @@
 import java.nio.ByteBuffer;
 
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
 
 import org.apache.geode.distributed.internal.DMStats;
+import org.apache.geode.util.internal.GeodeGlossary;
 
 public class BufferPoolTest {
 
+  @Rule
+  public final RestoreSystemProperties restorer = new RestoreSystemProperties();
```

Comment:
Using Junit5 and Pioneer system properties annotations will eliminate need for this legacy rule.